microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -340,6 +340,14 @@ Collecting child browsing data and sharing it with the family group organizer re
 The family group organizer may stop the data collection from the family safety portal.  For more information about Microsoft family safety features, see [What is a Microsoft family group?](https://support.microsoft.com/help/12413)
 
 
+<!-- ====================================================================== -->
+## Favorites
+
+Favorites now supports views that add an image thumbnail for favorites a user has saved.  All favorites data is stored locally on the device, and is organized per Microsoft Edge profile. Favorites are saved for the duration the favorite exists (10 years).  If you have sync turned on for favorites, your saved items are available across all signed-in and syncing versions of Microsoft Edge.
+
+If you want Microsoft Edge to not capture images for your favorites, go to `edge://settings/privacy`, scroll down to the **Services** section, and then turn off the button next to **Save Images for Favorites**.<!-- todo: no such control found in Canary 117 -->  All previously saved images will be erased.
```

Comment:
In Canary 117, in `edge://settings/privacy` > **Services** section, **Save Images for Favorites** is not found.

Reply:
Style guide (I might tweak UI text, after confirming the UI):
* "go to": https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/n/navigate
* "toggle": https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/t/toggle

Reply:
replaced by fresh comment https://github.com/MicrosoftDocs/edge-developer/pull/2760/files#r1283902474